_integration-test/ensure-backup-restore-works.sh

```diff
@@ -11,24 +11,24 @@ echo "Creating backup..."
 touch $(pwd)/sentry/backup.json
 chmod 666 $(pwd)/sentry/backup.json
 # Command here matches exactly what we have in our docs https://develop.sentry.dev/self-hosted/backup/#backup
-docker-compose run -v $(pwd)/sentry:/sentry-data/backup --rm -T -e SENTRY_LOG_LEVEL=CRITICAL web export /sentry-data/backup/backup.json
+$dc run -v $(pwd)/sentry:/sentry-data/backup --rm -T -e SENTRY_LOG_LEVEL=CRITICAL web export /sentry-data/backup/backup.json
```

Comment:
using `$dc` since our scripts do some pre-script setup that is failing CI (error-handling, dc-detect-version, detect-platform, trap)

Reply:
Yisss, exactly the cases we wanted to fix.